test/integration/functional_test.go

```diff
@@ -173,6 +175,46 @@ func validateComponentHealth(ctx context.Context, t *testing.T, profile string)
 	}
 }
 
+func validateStatusCmd(ctx context.Context, t *testing.T, profile string) {
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "status"))
```

Comment:
You'll need to pass `"-p", profile` to all of commands you are running, otherwise these tests only run against the `minikube` profile, which doesn't necessarily exist when run in parallel.

Reply:
@tstromberg I have made changes as suggested.